mongo/integration/unified/unified_spec_runner.go

```diff
@@ -114,22 +126,42 @@ func RunTestFile(t *testing.T, filepath string, opts ...*Options) {
 			CreateClient(false)
 
 		mt.RunOpts(testCase.Description, mtOpts, func(mt *mtest.T) {
-			if err := testCase.Run(mt); err != nil {
-				mt.Fatal(err)
+			defer func() {
+				// catch panics from looking up elements and fail if it's unexpected
+				if r := recover(); r != nil {
```

Comment:
We use a lot of panicking raw lookup/casts and I can alternatively go through and change all of those to use _OK methods, but that's a considerable change

Reply:
Hmm seems out of scope for this ticket to me, and `recover`ing the `panic` seems acceptable.